pkg/controller/controller.go

```diff
@@ -375,6 +387,12 @@ func (ctrl *resizeController) resizePVC(pvc *v1.PersistentVolumeClaim, pv *v1.Pe
 		pvc = updatedPVC
 	}
 
+	err := ctrl.claims.Update(pvc)
+	if err != nil {
+		klog.Errorf("error updating PVC %s in cache: %v", util.PVCKey(pvc), err)
```

Comment:
Can we return this error and let the caller log?

Reply:
fixed